src/main/java/uk/gov/ons/census/casesvc/utility/FieldworkHelper.java

```diff
@@ -3,17 +3,46 @@
 import uk.gov.ons.census.casesvc.model.entity.Case;
 
 public class FieldworkHelper {
-  public static boolean shouldSendCaseToField(Case caze, String eventChannel) {
-    if (eventChannel.equals("FIELD")) {
+  public static boolean shouldSendUpdatedCaseToField(Case caze, String eventChannel) {
+    if (hasChannelField(eventChannel)) {
       return false;
     }
-    if (caze.getRegion().startsWith("N") && caze.getCaseType().equals("CE")) {
+
+    if (hasRegionNorthernIrelandAndCECaseType(caze)) {
       return false;
     }
+
     if (caze.getEstabType().equals("TRANSIENT PERSONS")
         || caze.getEstabType().equals("MIGRANT WORKERS")) {
       return false;
     }
     return true;
   }
+
+  public static boolean shouldSendUnInvalidatedAddressCaseToField(Case caze, String eventChannel) {
```

Comment:
I'm not sure we can justify the proliferation of near-identical methods, with different names. I'm sure Dan said that the business logic for sending CREATE and UPDATE messages is identical. Only CANCEL has different logic.

Reply:
We will ask Dan about the acceptance criteria as it does seem odd that the UPDATE is similar to CREATE 👍